contributors/guide/help-wanted.md

```diff
@@ -102,6 +106,10 @@ labels (above) in addition to meeting the following criteria:
   fit to be copied. If the area of code doesn't have tests, before labeling the
   issue, add a test fixture. This prep often makes a great `help wanted` task!
 
+Example of a Good First Issue:
+  
+  [update doc.go in staging/ repositories kubernetes#68231](https://github.com/kubernetes/kubernetes/issues/68231)
+  
```

Comment:
Example of a Good First Issue description can be found here: [kubernetes/kubernetes#68231](https://github.com/kubernetes/kubernetes/issues/68231).